certified-connectors/HoneywellForge/apiProperties.json

```diff
@@ -0,0 +1,35 @@
+{
+  "properties": {
+    "capabilities": [],
+    "connectionParameters": {
+      "token": {
+        "type": "oauthSetting",
+        "oAuthSettings": {
+            "identityProvider": "oauth2",
+            "clientId": "YOUR_CLIENT_ID",
+            "clientSecret": "YOUR_CLIENT_SECRET",
+            "scopes": [
+              "openid forge.access offline"
+            ],
+            "redirectMode": "Global",
+            "redirectUrl": "https://global.consent.azure-apim.net/redirect",
+            "properties": {
+                "IsFirstParty": "False"
+            },
+            "customParameters": {
+                "authorizationUrl": {
+                    "value": "https://forge-sso.spec.honeywell.com/oauth2/token"
+                },
+                "tokenUrl": {
+                    "value": "https://forge-sso.spec.honeywell.com/oauth2/token"
+                },
+                "refreshUrl": {
+                    "value": "https://forge-sso.spec.honeywell.com/oauth2/token"
+                }
+            }
+        }
+      }
+    },
+    "iconBrandColor": "#EE4677"
```

Comment:
@ashishdotme You need to add `"publisher"` and `"stackOwner"` properties as well.

Documentation - [Add metadata](https://docs.microsoft.com/en-us/connectors/custom-connectors/certification-submission#step-4a-publisher-and-stackowner-properties)